frame/society/src/lib.rs

```diff
@@ -476,41 +476,41 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config<I>, I: 'static = ()> {
 		/// The society is founded by the given identity. \[founder\]
```

Comment:
```suggestion
		/// The society is founded by the given identity.
```

Same for all other event variants.

Reply:
Okay @KiChjang, will do 👍